exercises/say/example.go

```diff
@@ -1,45 +1,53 @@
 package say
 
+import "errors"
+
+const testVersion = 1
+
 var small = []string{"zero", "one", "two", "three", "four", "five", "six",
 	"seven", "eight", "nine", "ten", "eleven", "twelve", "thirteen",
 	"fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"}
 var tens = []string{"ones", "ten", "twenty", "thirty", "forty",
 	"fifty", "sixty", "seventy", "eighty", "ninety"}
-var scale = []string{"thousand", "million", "billion",
-	"trillion", "quadrillion", "quintillion"}
+var scale = []string{"thousand", "million", "billion"}
 
-func Say(n uint64) string {
+func Say(n uint64) (string, error) {
+	if n >= 1000000000000 {
+		return "", errors.New("value out of range")
+	}
 	switch {
 	case n < 20:
-		return small[n]
+		return small[n], nil
 	case n < 100:
 		t := tens[n/10]
 		s := n % 10
 		if s > 0 {
 			t += "-" + small[s]
 		}
-		return t
+		return t, nil
 	case n < 1000:
 		h := small[n/100] + " hundred"
 		s := n % 100
 		if s > 0 {
-			h += " " + Say(s)
+			temp, _ := Say(s)
```

Comment:
I'm not sure about this variable name, I understand it's to signify temporary, but I think `x` would suffice, the use of `temp` rather than a letter seems to infer more importance on it than is necessary. 

I also think this error should be handled. At this point in the function it shouldn't error out, but still it's better to handle it than ignore it I think.